client/my-sites/plans-grid/components/plan-type-selector/style.scss

```diff
@@ -1,3 +1,5 @@
+// TODO: Check if I need to update this import to avoid referencing calypso alias
```

Comment:
Thank you! I double checked this and it looks fine since there are `calypso` scss path aliases used throughout the `plans-grid` package.

Removing the inline comment now